coops-ui/src/views/StandaloneOfficeAddressFiling.vue

```diff
@@ -35,15 +35,28 @@
       <v-container id="standalone-office-address-container" class="view-container">
         <article id="standalone-office-address-article">
           <header>
-            <h1 id="filing-header">Address Change</h1>
+            <h1 id="filing-header">Change of Office Addresses</h1>
+            <p>Please change your Registered Office Address
+              <span v-if="entityFilter(EntityTypes.BCorp)">and Record Address</span>
+            </p>
+            <v-alert
+              v-if="entityFilter(EntityTypes.BCorp)"
+              type="info"
+              :value="true"
```

Comment:
Since value doesn't need to be reactive (ie, based on a property that changes), try this (ie, no colon):
```
value="true"
```

Reply:
oops, this may not even need to be there. I believe i just left this in while testing.

Reply:
oops, this may not even need to be there. I believe i just left this in while testing.

Reply:
Actually, you don't need value at all, since by default it's true. And while you're there, could you please fix this in StandaloneDirectorsFiling.vue as well? Oops and thanks.

Reply:
Sounds good, thanks!

Reply:
Fixed, thank you.